playbooks/roles/ocp-metallb-operator/tasks/e2e.yml

```diff
@@ -0,0 +1,1682 @@
+---
+# e2e testcases metallb-operator
+- debug:
+    msg: Print the value in vars {{ test_image }}
+
+- name: Validate the controller and speaker pods can be scheduled on the tainted nodes based on the tolerations in metallb CR
+  block:
+  - name: Check the number of nodes running
+    shell: oc get node | grep "Ready" | wc -l
+    register: metallb_nodes
+
+  - name: Taint master and worker nodes
+    shell: |
+      oc adm taint node master-0 master-group=infra:NoSchedule
```

Comment:
Can you ensure that the node names come from the cluster as node names may not always remain the same as master-0,worker-0 etc.